environments/skeleton/{{cookiecutter.environment}}/terraform/compute.tf

```diff
@@ -18,4 +18,6 @@ module "compute" {
   k3s_token = var.k3s_token
   control_address = [for n in openstack_compute_instance_v2.control["control"].network: n.fixed_ip_v4 if n.access_network][0]
   security_group_ids = [for o in data.openstack_networking_secgroup_v2.nonlogin: o.id]
+
+  compute_init_enable = var.compute_init_enable
```

Comment:
Nope, this means we can't define it per compute node group. It needs to be taken from each.value.